test_n13_api.py

```diff
@@ -36,11 +37,14 @@ def test_deflate(config):
     assert r.elapsed.total_seconds() < 0.500
 
 
-def test_gzip(config):
-    r = get(config['base_url'] + 'gzip')
+@pytest.mark.parametrize('input_connection','expected_connection',[
```

Comment:
Travis failed because you specified 2 strings here but it should be 1 string with coma-separated parameters names:
~'input_connection','expected_connection'~
'input_connection,expected_connection'
it's just FYI, but you need only 1 param - see my previous comment